graphql-server/migrations/1588309975119_alter_table_public_games_alter_column_num_entries_per_player/up.yaml

```diff
@@ -0,0 +1,11 @@
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE "public"."games" ALTER COLUMN "num_entries_per_player" TYPE int4;
+  type: run_sql
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE ONLY "public"."games" ALTER COLUMN "num_entries_per_player" SET
+      DEFAULT 6;
```

Comment:
![image](https://user-images.githubusercontent.com/2534903/80822950-90da5280-8b90-11ea-8b07-38c028a07dc1.png)

![image](https://user-images.githubusercontent.com/2534903/80822985-9b94e780-8b90-11ea-81fc-64558ccc34a0.png)

Reply:
cc: @wtachau I went back and forth b/w 4 and 6... and settled on 6 haha 😆